core/src/test/java/com/datastax/dse/driver/internal/core/cql/reactive/TestSubscriber.java

```diff
@@ -81,7 +81,8 @@ public List<T> getElements() {
   }
 
   public void awaitTermination() {
-    Uninterruptibles.awaitUninterruptibly(latch, 1, TimeUnit.MINUTES);
-    if (latch.getCount() > 0) fail("subscriber not terminated");
+    if (!Uninterruptibles.awaitUninterruptibly(latch, 1, TimeUnit.MINUTES)) {
+      fail("subscriber not terminated");
+    }
```

Comment:
This seems correct.  Uninterruptibles.awaitUninterruptibly() calls CountDownLatch.await() which (as I read the Javadoc) returns true if the latch count is equal to zero: in all other cases (including throwing an InterruptedException) execution continues until the timeout is hit which results in a return value of false.